Dockerfile

```diff
@@ -207,6 +207,12 @@ ARG LIBMYSOFA_SHA256=94cb02e488de4dc0860c8d23b29d93d290bb0a004d4aa17e1642985bba1
 # bump: uavs3d link "Source diff $CURRENT..$LATEST" https://github.com/uavs3/uavs3d/compare/$CURRENT..$LATEST
 ARG UAVS3D_URL="https://github.com/uavs3/uavs3d.git"
 ARG UAVS3D_COMMIT=57d20183301d4197d1c938f62f8a5911e33465d7
+# bump: rubberband /RUBBERBAND_COMMIT=([[:xdigit:]]+)/ gitrefs:https://github.com/breakfastquay/rubberband.git|re:#^refs/heads/master$#|@commit
+# bump: rubberband after ./hashupdate Dockerfile RUBBERBAND $LATEST
+# bump: rubberband after COMMIT=$(git ls-remote https://github.com/breakfastquay/rubberband.git $LATEST^{} | awk '{print $1}') && sed -i -E "s/^ARG RUBBERBAND_COMMIT=.*/ARG RUBBERBAND_COMMIT=$COMMIT/" Dockerfile
+# bump: rubberband link "Source diff $CURRENT..$LATEST" https://github.com/breakfastquay/rubberband/compare/$CURRENT..$LATEST
+ARG RUBBERBAND_URL="https://github.com/breakfastquay/rubberband.git"
+ARG RUBBERBAND_COMMIT="b3c920a35ed1ea4da37ddd62a12d3a81278097d1"
```

Comment:
Has no versioned releases?

Reply:
It seems so, so we can probably do something similar to the ffmpeg bump config. I can fix it if you want.